server/routes/incentiveLevels.ts

```diff
@@ -12,10 +12,6 @@ import IncentiveLevelEditForm from './forms/incentiveLevelEditForm'
 import IncentiveLevelReorderForm from './forms/incentiveLevelReorderForm'
 import IncentiveLevelStatusForm from './forms/incentiveLevelStatusForm'
 
-export function sanitiseIncentiveName(name: string) {
-  return name.trim()
-}
-
```

Comment:
I've moved the sanitisation responsibility into the relevant forms. It seems sensible to encapsulate the forms actions within the relevant class rather than rely on the careful use of the form in each situation.